sql/src/main/java/io/crate/action/sql/RowReceiverToResultReceiver.java

```diff
@@ -27,7 +27,7 @@
 
 import java.util.Set;
 
-class RowReceiverToResultReceiver implements RowReceiver {
+public class RowReceiverToResultReceiver implements RowReceiver {
```

Comment:
This class shouldn't become public. Please move into other package if required to keep it package local